src/robot/libraries/Process.py

```diff
@@ -667,6 +680,19 @@ def get_process_object(self, handle=None):
         """
         return self._processes[handle]
 
+    def get_process_object_index(self, process=None):
+        """Returns handle (index of object in connection cache) for given process object.
+
+        If ``process`` is not given, uses the current `active process`.
+
+        Examples:
+        | Terminate All Processes |
+        | ${process_object}=      | Start Process | ${command} |
+        | ${process_index}=       | Get Process Object Index   | ${process_object} |
+        | Should Be Equal         | ${process_index}           | ${1}
+        """
+        return self._processes.get_connection_index(process)
```

Comment:
I don't think anyone needs the old process handle (i.e. index) for anything. Apparently this keyword was added to make it fix a test that was broken when `Start Process` doesn't anymore return indices, but I already commented that there are other ways to validate that the internal cache is empty.

In other words, I believe this keyword should be removed. Can be added later if someone actually needs it.

Reply:
got rid of `get_connection_index` method